misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java

```diff
@@ -2277,20 +2345,93 @@ public RestArgumentNode transform(RestArgumentNode restArgumentNode) {
 
     @Override
     public ObjectTypeDescriptorNode transform(ObjectTypeDescriptorNode objectTypeDescriptorNode) {
+        NodeList<Token> objectTypeQualifiers = formatNodeList(objectTypeDescriptorNode.objectTypeQualifiers(),
+                1, 0, 1, 0);
+        Token objectKeyword = formatToken(objectTypeDescriptorNode.objectKeyword(), 1, 0);
+        Token openBrace = formatToken(objectTypeDescriptorNode.openBrace(), 0, 1);
+        indent();
+        NodeList<Node> members = formatNodeList(objectTypeDescriptorNode.members(), 0, 1, 0, 1);
+        unindent();
+        Token closeBrace = formatToken(objectTypeDescriptorNode.closeBrace(), this.trailingWS, this.trailingNL);
```

Comment:
We might have to set indentation to the braces as well, similar to the record-type-desc